docs/books/computer-games/computer-battle-games/robot_missile/robot_missile.py

```diff
@@ -1 +1,31 @@
+import random
+import string
 
+print('ROBOT MISSLE' \
+      '\n'
+      '\nType the correct code letter (A-Z) to defuse the missle.' \
```

Comment:
Those missiles keep missing their 'i's... :-)

Reply:
dang it keyboard